### DISCUSSION TITLE
Is Apache `superset` a better dashboard interface to the `pyani` database than `flask`?

### DISCUSSION THREAD

**Discussion by widdowquinn** [widdowquinn/pyani#314] [Ideas]
We should investigate whether this is a useful view onto the `pyanidb` database.

- `superset` repo: https://github.com/apache/superset

**Comment by baileythegreen** [comment]
@widdowquinn We don't currently use either, right? So this is just deciding on the better option to implement?

**Comment by widdowquinn** [comment]
Yes. It's a reminder to check it out when we get to implementing some kind of visual UI onto the results database.

**Reply by baileythegreen** [reply]
This was a reply to the comment:

> @widdowquinn We don't currently use either, right? So this is just deciding on the better option to implement?

**Reply by widdowquinn** [reply]
Yes. I know flask, but I don't know superset. Inertia etc. would lead me to Flask as the simplest option - but that's not a compelling reason to use it.
